test/contrib/isotp_soft_socket_fd.uts

```diff
@@ -0,0 +1,1056 @@
+% Regression tests for ISOTPSoftSocket with CanFD enabled
```

Comment:
It seems unnecessary to copy/paste all the tests and just tweak a settings.
It's usually a better practice to keep just a few tests that really do test a meaningfully different feature, but are maintainable.

Reply:
Thanks for you Feedback. Done!